ci/authn-azure/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
```

Comment:
`-ex` are a couple of options that gives a sort of granularity in terms of failures and logs. I know there are `o, a`, etc but what does `-ex` specifically represent?